flight/mcu/lib/teensyShared/dataLog.h

```diff
@@ -54,6 +55,11 @@ struct GpsData {
     bool posLock;
     systime_t timeStamp_GPS;
 };
+struct flapData {
```

Comment:
flapData should be upper camel case
contents of flap data should be updated for the new flap configuration